pycbc/inference/recal.py

```diff
@@ -0,0 +1,102 @@
+# Module to hold recalibration stuff used during parameter estimation
+# This will draw heavily from pycbc_adjust_strain and cal.py which
+# are part of Chris Biwer's pycbc-cal repository
+
+import numpy
+from scipy.interpolate import UnivariateSpline
+from pycbc.types import FrequencySeries
+
+class Recalibrate(object):
+    """ Class for adjusting time-varying calibration parameters.
+    """
```

Comment:
This needs more documentation on how the class is initialized. See examples in ``pycbc.distribtions`` or ``pycbc.inference`` for the kinds of things we include in these descriptions.